script.js

```diff
@@ -25,4 +25,216 @@ closeLinks.forEach((closeLink) => {
+      projHeader.setAttribute('class', 'proj-header');
+      projHeader.textContent = projectinfo[0].pojectH2;
+
+      const headerlinkingList = document.createElement('ul');
+      headerlinkingList.setAttribute('class', 'headerlink-lists');
+      for (let e = 0; e < projectinfo[0].links.length; e += 1) {
+        const headerlinkItem = projectinfo[0].links[e];
+
+        const headerlinkList = document.createElement('li');
+        headerlinkList.setAttribute('class', 'headerlink-list');
+
+        const headerlinkElement = document.createElement('a');
+        headerlinkElement.setAttribute('href', headerlinkItem.link);
+
+        const headerlinkText = document.createElement('span');
+        headerlinkText.textContent = headerlinkItem.text;
+        headerlinkElement.appendChild(headerlinkText);
+
+        const headerlinkIcon = document.createElement('i');
+        headerlinkIcon.setAttribute('class', headerlinkItem.Icon);
+        headerlinkElement.appendChild(headerlinkIcon);
+
+        headerlinkList.appendChild(headerlinkElement);
+        headerlinkingList.appendChild(headerlinkList);
+      }
+
+      projHeaderdiv.appendChild(projHeader);
+      projHeaderdiv.appendChild(headerlinkingList);
+
+      const listElement = document.createElement('ul');
+      listElement.setAttribute('class', 'tech-list');
+      for (let r = 0; r < projectinfo[0].technologies.length; r += 1) {
+        const item = projectinfo[0].technologies[r];
+        const itemElement = document.createElement('li');
+        itemElement.textContent = item;
+        listElement.appendChild(itemElement);
+      }
+
+      const paragraph = document.createElement('p');
+      paragraph.setAttribute('class', 'proj-paragraph');
+      paragraph.textContent = projectinfo[0].description;
+
+      const linkingList = document.createElement('ul');
+      linkingList.setAttribute('class', 'link-lists');
+      for (let s = 0; s < projectinfo[0].links.length; s += 1) {
+        const linkItem = projectinfo[0].links[s];
+
+        const linkList = document.createElement('li');
+        linkList.setAttribute('class', 'link-list');
+
+        const linkElement = document.createElement('a');
+        linkElement.setAttribute('href', linkItem.link);
+
+        const linkText = document.createElement('span');
+        linkText.textContent = linkItem.text;
+        linkElement.appendChild(linkText);
+
+        const linkIcon = document.createElement('i');
+        linkIcon.setAttribute('class', linkItem.Icon);
+        linkElement.appendChild(linkIcon);
+
+        linkList.appendChild(linkElement);
+        linkingList.appendChild(linkList);
+      }
+
+      mainDiv.appendChild(closebtnDiv);
+      mainDiv.appendChild(imgContainer);
+      mainDiv.appendChild(projHeaderdiv);
+      mainDiv.appendChild(listElement);
+      mainDiv.appendChild(paragraph);
+      mainDiv.appendChild(linkingList);
+      popupWindow.appendChild(mainDiv);
```

Comment:
- [ ] At the moment, when you open a project modal for the second time onwards it adds new modal to the already opened modal. Which means the second is two modals in one, third is three and so on. Please kindly fix this.

![image](https://user-images.githubusercontent.com/43995830/216063153-e0c61d9c-8307-4f07-8aba-701747a74d40.png)